WebRoot/public/login.jsp

```diff
@@ -587,14 +587,14 @@ if (application.getInitParameter("offlineMode") != null) {
                                 <input type="submit" class="ZLoginButton DwtButton" value="<fmt:message key="login"/>" />
                                 </td>
                                 </tr>
-                                <c:if test="${domainInfo.attrs.zimbraFeatureResetPasswordStatus eq 'enabled'}">
+                                <%-- <c:if test="${domainInfo.attrs.zimbraFeatureResetPasswordStatus eq 'enabled'}"> --%>
```

Comment:
Why comment this out instead of remove it? Also doesn't this force this section to appear even when the reset password feature is disabled?

Reply:
I was asked to remove and open this up.

Reply:
Shouldn't you remove it then?